src/process/dust/CCPr_Dust_Mod.F90

```diff
@@ -237,13 +237,53 @@ SUBROUTINE CCPr_Dust_Run( MetState, DiagState, DustState, ChemState, RC )
          ! Run the Dust Scheme
          !--------------------
          if (DustState%SchemeOpt == 1) then ! FENGSHA
-            call CCPr_Scheme_Fengsha( MetState, DiagState, DustState, RC )
+            call CCPr_Scheme_Fengsha(DustState%nDustSpecies,          &
+               MetState%DSOILTYPE,              &
+               MetState%SSM,                    &
+               MetState%RDRAG,                  &
+               MetState%TSKIN,                  &
+               MetState%USTAR,                  &
+               MetState%USTAR_THRESHOLD,        &
+               MetState%GWETTOP,                &
+               MetState%z0,                     &
+               MetState%CLAYFRAC,               &
+               MetSTate%SANDFRAC,               &
+               MetState%AIRDEN(1),              &
+               MetState%FROCEAN,                 &
+               MetState%FRLANDIC,               &
+               MetState%FRSNO,                  &
+               DustState%AlphaScaleFactor,      &
+               DustState%BetaScaleFactor,       &
```

Comment:
its gamma in the subroutine, should it be gamma here instead of beta?

Reply:
🤷

Reply:
🤷‍♂️